bin/commands/start.js

```diff
@@ -4,136 +4,110 @@ var
   fs = require('fs'),
   rc = require('rc'),
   params = rc('kuzzle'),
-  kuzzle = require('../../lib'),
+  //kuzzle = require('../../lib'),
+  Kuzzle = require('../../lib/api/kuzzle'),
   RequestObject = require('kuzzle-common-objects').Models.requestObject,
   Promise = require('bluebird'),
   clc = require('cli-color'),
-  coverage;
+  coverage,
+  kuzzleLogo = `
+      ▄▄▄▄▄      ▄███▄      ▄▄▄▄
+   ▄█████████▄▄█████████▄▄████████▄
+  ██████████████████████████████████
+   ▀██████████████████████████████▀
+    ▄███████████████████████████▄
+  ▄███████████████████████████████▄
+ ▀█████████████████████████████████▀
+   ▀██▀        ▀██████▀       ▀██▀
+          ██     ████    ██
+                ▄████▄
+                ▀████▀
+                  ▀▀`;
 
 module.exports = function (options) {
   var
+    kuzzle = new Kuzzle(),
     error = string => options.parent.noColors ? string : clc.red(string),
     warn = string => options.parent.noColors ? string : clc.yellow(string),
     notice = string => options.parent.noColors ? string : clc.cyanBright(string),
     ok = string => options.parent.noColors ? string : clc.green.bold(string),
     kuz = string => options.parent.noColors ? string : clc.greenBright.bold(string);
-  
+
   if (process.env.FEATURE_COVERAGE === '1' || process.env.FEATURE_COVERAGE === 1) {
     coverage = require('istanbul-middleware');
     console.log(warn('Hook loader for coverage - ensure this is not production!'));
     coverage.hookLoader(__dirname+'/../lib');
   }
+  console.log(kuz('Starting Kuzzle'), (options.server ? notice('Server') : warn('Worker')));
```

Comment:
to be changed to
`console.log(kuz('Starting Kuzzle'), notice('Server'));`
or just `console.log(kuz('Starting Kuzzle'));`